<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::LeftToRight</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="icon">
         <iconset resource="derfla.qrc">
          <normaloff>:/rc/images/general.png</normaloff>:/rc/images/general.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Extension</string>
        </property>
        <property name="icon">
         <iconset resource="derfla.qrc">
          <normaloff>:/rc/images/extension.png</normaloff>:/rc/images/extension.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Plugin</string>
        </property>
        <property name="icon">
         <iconset resource="derfla.qrc">
          <normaloff>:/rc/images/plugin.png</normaloff>:/rc/images/plugin.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Skin</string>
        </property>
        <property name="icon">
         <iconset resource="derfla.qrc">
          <normaloff>:/rc/images/skin.png</normaloff>:/rc/images/skin.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="generalPage"/>
       <widget class="QWidget" name="extensionPage"/>
       <widget class="QWidget" name="pluginPage"/>
       <widget class="QWidget" name="skinPage"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="derfla.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
